syntax/openscad.vim

```diff
@@ -28,7 +28,7 @@ syn keyword openscadPrimitiveSolid cube sphere cylinder polyhedron surface
 syn keyword openscadPrimitive2D square circle polygon import_dxf text
 syn keyword openscadPrimitiveImport import child children
 
-syn match openscadSpecialVariable "\$[a-zA-Z]\+\>" display
+syn match openscadSpecialVariable "\$[a-zA-Z0-9_]\+\>" display
```

Comment:
This will match all tokens like "$1234" as well. Is this intended?

If not, maybe something like "[a-zA-Z_][a-zA-Z0-9_]+" will work better.

Reply:
I tried this:

```scad
$23 = 400;
cube($23);
```

and it works fine, so the regex from the OpenSCAD docs used in this PR is fine.